### PR TITLE
Fix update NetAttDefs in different namespaces

### PR DESCRIPTION
**Pull request by oglok**
Instantiate a new client to avoid querying the cache which
  won't find any NetAttDef in a different namespace

Signed-off-by: Ricardo Noriega <rnoriega@redhat.com>

### CODE REVIEW

File: pkg/controller/sriovnetwork/sriovnetwork_controller.go [hunk 1]
```diff
@@ -41,12 +44,33 @@ var log = logf.Log.WithName("controller_sriovnetwork")
 // Add creates a new SriovNetwork Controller and adds it to the Manager. The Manager will set fields on the Controller
 // and Start it when the Manager is Started.
 func Add(mgr manager.Manager) error {
-	return add(mgr, newReconciler(mgr))
+
```
Comment (pliurh) [inline pkg/controller/sriovnetwork/sriovnetwork_controller.go]: nit: remove blank line.

File: pkg/controller/sriovnetwork/sriovnetwork_controller.go [hunk 2]
```diff
@@ -41,12 +44,33 @@ var log = logf.Log.WithName("controller_sriovnetwork")
...
+	}
+
+	return add(mgr, reconciler)
 }
 
 // newReconciler returns a new reconcile.Reconciler
-func newReconciler(mgr manager.Manager) reconcile.Reconciler {
-	return &ReconcileSriovNetwork{client: mgr.GetClient(), scheme: mgr.GetScheme()}
+func newReconciler(mgr manager.Manager) (reconcile.Reconciler, error) {
+
```
Comment (pliurh) [inline pkg/controller/sriovnetwork/sriovnetwork_controller.go]: nit: remove blank line.

File: pkg/controller/sriovnetwork/sriovnetwork_controller.go (hunk 1 above)
Comment (oglok) [reply pkg/controller/sriovnetwork/sriovnetwork_controller.go]: Ack

File: pkg/controller/sriovnetwork/sriovnetwork_controller.go (hunk 2 above)
Comment (oglok) [reply pkg/controller/sriovnetwork/sriovnetwork_controller.go]: Ack

**Pull request merged**